packages/audit/hooks/audit-route.py

```
@@ -0,0 +1,257 @@
+    return None, "", tool_name
+
+
+def is_match(expected_route: str, actual_route: str, aliases: dict) -> bool:
+    """予測ルートと実ルートが一致するか判定する。
+
+    マッチ条件:
+    - 完全一致
+    - aliases[expected_route] に actual_route が含まれる
+
+    `skill:*` も aliases 経由で許可する必要があり、ポリシー未定義の skill は
+    不一致扱いとなる（過剰なマッチを避けるため）。
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)